README.md

```diff
@@ -6,6 +6,13 @@
 
 ## News
 
+Argo Project is moving from CLA to DCO as a more contributor friendly way to track contributions.
```

Comment:
how is this more friendly?

Reply:
The DCO is much shorter to read and understand.